public/js/controllers/traffic_graphs_controller.js

```diff
@@ -219,7 +251,7 @@ export class TrafficGraphsController {
         let receivedSliderStep = d3
             .sliderLeft()
             .min(0)
-            .max(yLimitReceived + (0.2 * yLimitReceived))
+            .max(yLimitReceived + 0.2 * yLimitReceived)
```

Comment:
```suggestion
            .max(yLimitReceived + (0.2 * yLimitReceived))
```
use nested parenthesis here to separate the math equation